tests/testthat/test-tidyme.R

```diff
@@ -0,0 +1,86 @@
+#' @title tidyme
+#' @section Last updated by:
+#' Tim Treis
+#' @section Last update date:
+#' 19-MAR-2021
+
+# Specifications ---------------------------------------------------------------
+
+#' T1.  The function properly converts "messy output" into tidy tibbles
+#' T1.1 The `visR::tidyme(stats::lm())` transformation doesn't change the values
+#' T1.2 The `visR::tidyme(survival::survfit())` transformation doesn't change the values
+#' T2.  Warnings and errors are thrown when neccecary
+#' T2.1 No error when a `stats::lm()` model is transformed
+#' T2.2 No error when a `survival::survfit()` model is transformed
+#' T2.3 Message when a non-survfit object is transformed
+
+# Requirement T1 ---------------------------------------------------------------
+
+context("tidyme - The function properly converts \"messy output\" into tidy tibbles")
+
+testthat::test_that("T1.1 The `visR::tidyme(stats::lm())` transformation doesn't change the values",{
+  
+  lm_object <- stats::lm(data = adtte, TRTDUR ~ AVAL)
```

Comment:
I would compare tidy::broom(lm) with tidyme(lm). Is much easier, but on the other hand your approach is much more robust.

Reply:
Then I'd just stick with the more robust one?